opm/material/fluidsystems/BrineCO2FluidSystem.hpp

```diff
@@ -305,7 +306,8 @@ class BrineCO2FluidSystem
         }
 
         assert(phaseIdx == gasPhaseIdx);
-        LhsEval result = CO2::gasViscosity(temperature, pressure);
+        CO2Tables params; // TODO: avoid this initialization
```

Comment:
I agree with the TODO...

We must find a solution to this (all of the similar TODO places here) before it can be merged.

Reply:
Resolved with a static singleton. This will not cause problems for the GPUification of the code as Flow does not use this fluid system regardless.